persistence/src/main/scala/com/evolutiongaming/akkaeffect/persistence/EventSourcedActorOf.scala

```diff
@@ -0,0 +1,135 @@
+package com.evolutiongaming.akkaeffect.persistence
+
+import akka.actor.Actor
+import akka.persistence.SnapshotSelectionCriteria
+
+import cats.effect.implicits.effectResourceOps
+import cats.effect.{Async, Resource, Ref}
+import cats.syntax.all._
+
+import com.evolutiongaming.akkaeffect._
+import com.evolutiongaming.catshelper.ToFuture
+
+import scala.reflect.ClassTag
+import java.time.Instant
+
+object EventSourcedActorOf {
```

Comment:
This object (or an `#actor` method?) may benefit from a scaladoc, which tells what is the main purpose of the factory.

I.e. it creates an `Actor`, which is able to accept messages of type `C`, perform a business logic described by  `EventSourcedOf` and ensure the events/snapshots are persisted using `EventSourcedPersistence`.